Задание_3/Задание_3/Program.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Задание_3
+{
+    internal class Program
+    {
+
+
+        static void Main()
+        {
+            double x, x3, x2, Y, A;
+            Console.WriteLine("Введите значение x");
+            x = Convert.ToDouble(Console.ReadLine());
+            x3 = 0;
+            x2 = 0;
+            Y = 0;
+            A = 0;
+            nomber(x, ref x3, ref x2, ref Y, ref A);
```

Comment:
Методы в C# принято именовать с большой буквы. Имя метода тоже желательно осмысленное. Передача по ссылке в данной ситуации не оправдана. Нужно просто передать в метод `x` и получить значение функции.
```suggestion
            y = Nomber(x);
```